invenio_pages/templates/invenio_pages/administration/page-search.html

```diff
@@ -0,0 +1,13 @@
+{#
+  Copyright (C) 2022 CERN.
```

Comment:
Remember to update the high end of the ranges in the files you modify.
```suggestion
  Copyright (C) 2023 CERN.
```